internal/services/redis/validate/access_keys_auth.go

```diff
@@ -0,0 +1,18 @@
+package validate
+
+import (
+	"fmt"
+	"log"
+)
+
+// Entra (AD) auth has to be set to disable access keys auth
+// https://learn.microsoft.com/en-us/azure/azure-cache-for-redis/cache-azure-active-directory-for-authentication
+func ValidateAccessKeysAuth(accessKeysAuthenticationDisabled bool, activeDirectoryAuthenticationEnabled bool) error {
```

Comment:
Since the logic here is simple enough, can we inline these to the CustomizeDiff instead, with the comment kept?